src/FMData.Rest/IAuthTokenProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace FMData.Rest
+{
+    /// <summary>
+    /// FileMaker REST Client Auth Provider Interface.
+    /// </summary>
+    public interface IAuthTokenProvider
+    {
+        /// <summary>
+        /// Connection config values
+        /// </summary>
+        ConnectionInfo Conn { get; }
```

Comment:
I'd like this to be `ConnectionInfo ConnectionInfo { get; }` to be inline with the rest of the naming conventions.